KeePassHTTPKit/Handlers/KPHHandler.m

```diff
@@ -106,7 +106,7 @@ - (NSArray *)entriesForURL:(NSURL *)url {
   /* if we got no scheme, host is nil as well so fall back to the full url string */
   NSString *search = url.host ? url.host : url.absoluteString;
   NSArray *entries = nil;
-  while (!entries || entries.count == 0) {
+  while ((!entries || entries.count == 0) && [search rangeOfString:@"."].length > 0) {
```

Comment:
This prevents "localhost" from ever matching which might be a use-case for some.